rfcs/0078-configuration-file-generators.md

```diff
@@ -123,7 +126,15 @@ widespread use of the module system inside `pkgs/`. There is also no guarantee
 that all the configuration files describing interaction of multiple software
 packages will have a clear choice of reference package.
 
-There also have been many solutions proposed based on a significant rework of the module system.
+A complete or partial merge of the module collections of the major currently
+existing module system based code bases, with internal options for platform
+specific implementation details. Unfortunately, this would force more coupling
+of the development cycle for the platform-specific parts. It also does not
+create a clean inspection/override point, and mixes the code with different
```

Comment:
Just a matter of calling `lib.evalModules`, `nix repl '<nixpkgs/nixos>'`, `arion repl` or whatnot, where the latter variations have the advantage of not having to decipher the logic that produces the inputs. Using options for this is an inspectability improvement over the usual `let` bindings and such that will be used with the function approach.